### DISCUSSION TITLE
Removing an App

### DISCUSSION THREAD

**Discussion by cvazquez** [codemagic-ci-cd/codemagic-docs#922] [Q&A] [answered]
How do I delete an app? I accidently created a duplicate and haven't set anything up in it. I just want to remove it. It's in my Personal Team section.

**Comment by mikhail-tokarev** [comment, marked as answer]
in `App Settings` (⚙️ icon) > `Repository settings` you can find buttons to archive and delete application

**Reply by cvazquez** [reply]
Thank you. Of course right after I submitted this, I found it on my own too 🙂
